src/components/App/App.js

```diff
@@ -9,24 +9,56 @@ class App extends Component {
   constructor() {
     super()
     this.state = {
-      currentPage: 'menu'
+      currentPage: 'landing',
+      landingScroll: '',
     }
   }
 
+  componentDidMount() {
+    const fetchtitleScroll = async() => {
+      const response = await fetch('https://swapi.co/api/films/')
+      const films = await response.json();
+      const randomNumber = Math.round(Math.random() * 7)
+      const film = films.results[randomNumber]
+      this.setState({ 
+        landingScroll: {
+          title: film.title,
+          year: film.release_date,
+          text: film.opening_crawl
+        }
+      })
+    }
+    fetchtitleScroll()
+    
+    setTimeout(() => {
```

Comment:
Could probably make componentDidMount async and not have to have another function (and call) inside of it. Then we could use Promise.race() to handle the timeout error.